Tools/autotest/sim_vehicle.py

```diff
@@ -121,95 +130,97 @@ def kill_tasks():
 # clean up processes at exit:
 atexit.register(kill_tasks)
 
+
 def check_jsbsim_version():
-    '''assert that the JSBSim we will run is the one we expect to run'''
+    """Assert that the JSBSim we will run is the one we expect to run"""
     jsbsim_cmd = ["JSBSim", "--version"]
     progress_cmd("Get JSBSim version", jsbsim_cmd)
     try:
         jsbsim_version = subprocess.Popen(jsbsim_cmd, stdout=subprocess.PIPE).communicate()[0]
     except OSError as e:
-        jsbsim_version = '' # this value will trigger the ".index"
-                            # check below and produce a reasonable
-                            # error message
+        jsbsim_version = ''  # this value will trigger the ".index"
+        # check below and produce a reasonable
```

Comment:
This indenting isn't so great.  The tabs in the original were wrong, but I think the concept is right (which obviously puts me at odds with the tool used to create the diff...